server/moodle/mod/livequiz/classes/services/livequiz_services.php

```diff
@@ -203,6 +191,7 @@ private function insert_question_with_relations(question $question, int $lecture
      * @throws Exception
      */
     private function submit_answers(int $questionid, array $answers): void {
+        $answerrepository = new answer_repository();
```

Comment:
Maybe make this into an instance variable as it is used in multiple places and the object is not changed. Maybe do the same for the other repositories